modules/ROOT/pages/index.adoc

```diff
@@ -127,6 +127,8 @@ When you deploy an application in Anypoint Runtime Fabric on, the following occu
 
 image::rtf-app-deployment.png[Application deployment workflow for Runtime Fabric]
 
+Application pods in the worker namespace must communicate with the resource cache in the Runtime Fabric agent namespace to retrieve deployable artifacts.
```

Comment:
You might want to reverse the order of this to emphasize the purpose of the info.
```suggestion
To retrieve deployable artifacts, application pods in the worker namespace must communicate with the resource cache in the Runtime Fabric agent namespace.
```